textx/scoping/rrel.py

```diff
@@ -56,7 +56,8 @@ class RRELBase(object):
     def __init__(self):
         pass
 
-    def get_next_matches(self, obj, lookup_list, allowed, first_element=False):
+    def get_next_matches(self, obj, lookup_list, allowed, matched_path_so_far,
```

Comment:
I propose shortening `matched_path_so_far` -> `matched_path` in all places. It is easier to read/write.

Reply:
(done)